src/main/java/com/powsybl/openloadflow/network/impl/AbstractLfBranch.java

```diff
@@ -245,10 +245,11 @@ public double computeApparentPower2() {
 
     @Override
     public boolean isZeroImpedanceBranch(boolean dc) {
+        double lowImpedanceThreshold = new LfNetworkParameters().getLowImpedanceThreshold();
```

Comment:
Same remark: you cannot create new LfNetworkParameters, you have to carry them here.

Reply:
Indeed you only have to carry the threshold, but the good one.